Userland/Libraries/LibJS/Runtime/JSONObject.h

```diff
@@ -18,13 +18,20 @@ class JSONObject final : public Object {
     virtual void initialize(GlobalObject&) override;
     virtual ~JSONObject() override;
 
+    FunctionObject& intrinsic_parse_function() const
+    {
+        return *m_intrinsic_parse_function;
+    }
+
```

Comment:
This doesn't feel great but I couldn't find a similar example how we do this.
So if there is a better way let me know!